client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -594,10 +609,6 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 			this.renderForm();
 		}
 
-		if ( this.props.plansFirst && ! this.props.selectedPlan ) {
-			return this.renderPlansSelector();
-		}
-
```

Comment:
Thanks for catching that! In 6c43dc6 I replaced a statically set prop for it's value, and cleaned up the related logic which led to these changes, but more 👀  is good.